Add Raspberry Pi Getting Started guide (#101)
Sensirion/embedded-sgp: docs/sgp40/raspberrypi/README.md

# SGP40 on Raspberry Pi

This document explains how to set the SGP40 up to run with a Raspberry Pi. This became very easy with the new SGP40 since it can run with a logic level of 3.3v which matches the one the Raspberry Pi has. 

> ## Getting started

Reviewer: maybe

```suggestion
## Setup Guide
```

### Connecting the Sensor

> Your Sensor should have four different connectors: VCC, GND, SDA, SCL. Please refere to other documentation of the Sensor to find out which connector is which on your specific modelle.

Reviewer: ```suggestion
Your sensor has the four different connectors: VCC, GND, SDA, SCL.
```

Use the following pins to connect your SGP40:

| *SGP40*        | *Raspberry Pi*  | 
|----------------|-----------------|

> | VCC            | Pin 1           |

Reviewer: could you add a description of the pin to make it easier to understand when using a different board, e.g.

```suggestion
| VCC            | Pin 1  (3.3V source)  |
```

| GND            | Pin 6           |
| SDA            | Pin 3           |
| SCL            | Pin 5           |

![RaspberryPi Pinout](https://www.raspberrypi.org/documentation/usage/gpio/images/GPIO-Pinout-Diagram-2.png)

###  Raspberry Pi

> - Install the newest Raspberry Pi OS on to your Raspberry Pi. If you don't know how to do that, there is a guide on the Raspberry Pi website (https://projects.raspberrypi.org/en/projects/raspberry-pi-setting-up).

Reviewer: ```suggestion
- Install the newest Raspberry Pi OS on to your Raspberry Pi. For help with this, kindly refer to the Raspberry Pi website (https://projects.raspberrypi.org/en/projects/raspberry-pi-setting-up).
```

- Enable the I2C interface in the Raspberry Pi config. 
  - Open a terminal.
  - Run `sudo raspi-config`.
  - Select "5 Interfacing Options  Configure connections to peripherals".
  - Select "P5 I2C  Enable/Disable automatic loading of I2C kernel module".
  - Select "Yes" when questioned "Would you like the ARM I2C interface to be enabled?".
  - The Raspberry Pi should respond with "The ARM I2C interface is enabled". Confirm with "Ok"
- Download driver.

>   - Go to the Sensirion SGP Driver Release page (https://github.com/Sensirion/embedded-sgp/releases) and download the `sgp40-VERSION.zip` file.

Reviewer: ```suggestion
  - Go to the Sensirion SGP Driver Release page (https://github.com/Sensirion/embedded-sgp/releases) and download the latest `sgp40-VERSION.zip` file, whereas `VERSION` represents the latest version.
```

  - Unzip the file into the directory where you want to install the driver (Run: `unzip sgp40-VERSION.zip` ). 
- Adapt the driver to work with Raspberry Pi.

>   - We use the linux_user_space implementation of the hw_i2c to run the driver on the Raspberry Pi. To use the in the project provided sample implementation navigate to the `./hw_i2c/sample-implementations/linux_user_space` direcory.

Reviewer: ```suggestion
  - We use the `linux_user_space` implementation from `hw_i2c` (The Linux kernel provides an I2C controller, thus "hardware" i2c) to run the driver on the Raspberry Pi. To use the in the project provided sample implementation navigate to the `./hw_i2c/sample-implementations/linux_user_space` directory.
```
